### ISSUE TITLE
Another vignette build failure

### ISSUE THREAD

**Issue by znmeb** [beanumber/openWAR#85]
```
> devtools::install_github('beanumber/openWAR', build_vignettes=TRUE)
Downloading github repo beanumber/openWAR@master
Installing openWAR
Skipping 2 packages not available: openWARData, Sxslt
'/usr/lib64/R/bin/R' --no-site-file --no-environ --no-save --no-restore CMD  \
  build '/tmp/RtmpIusTHG/devtools1a44ed5923d/beanumber-openWAR-b234a6b'  \
  --no-resave-data --no-manual 

* checking for file ‘/tmp/RtmpIusTHG/devtools1a44ed5923d/beanumber-openWAR-b234a6b/DESCRIPTION’ ... OK
* preparing ‘openWAR’:
* checking DESCRIPTION meta-information ... OK
* installing the package to build vignettes
* creating vignettes ... ERROR
Quitting from lines 101-102 (intro.Rmd) 
Error: processing vignette 'intro.Rmd' failed with diagnostics:
figure margins too large
Execution halted
Error: Command failed (1)
> 
```

**Comment by beanumber** [comment]
OK, I broke this, but I'm not sure what happened here. Note that:

```
library(openWAR)
data(May14)
class(May14)
plot(May14)
```
However, if you then say:
```
May14 %>%
  filter(event == "Home Run") %>%
  select(gameId, batterId, description)
class(May14)
plot(May14)
```
you get the error. The crazy part is that even though we didn't do anything to the `May14` object, somehow we changed it's class so that it is no longer a `GameDayPlays` object. Thus, the `plot()` method that is throwing the error is not `plot.GameDayPlays()`. 

This seems like a bug in `dplyr`. Maybe this is relevant (https://github.com/hadley/dplyr/issues/665)?

Any thoughts on why this is happening?
